test/run_test.py

```diff
@@ -317,6 +317,19 @@ def skip_test_p(name: str) -> bool:
 
 DISTRIBUTED_TESTS = [test for test in TESTS if test.startswith("distributed")]
 
+
+def discover_functorch_tests():
```

Comment:
Some of our linters assume that all tests are in the test folder (see our test ownership lint). Would it be bad to move functorch tests to be test/functorch?

Reply:
We can't move all functorch tests from functorch/test to the test/ folder yet. because we are still syncing functorch to the pytorch/functorch repository.

We're planning on stopping that sync in around a month, at which point we can actually move functorch tests from functorch/test to the test folder if that would make things better. Is this OK?

Also, it sounds like I should add some test ownership comments to the functorch tests?

Reply:
in the latest update I added some test owner comments to the test files

Reply:
can we have an issue to track moving functorch tests under test/ so we don’t forget about it?

Reply:
https://github.com/pytorch/pytorch/issues/82109